cli/runLambda.sh

```diff
@@ -52,6 +55,9 @@ function options() {
             l)
                 INCLUDE_LOG_TAIL="true"
                 ;;
+            g)
```

Comment:
I don't see a corresponding changes to getopts?

Reply:
Fixed